cozmo_soar.py

```diff
@@ -799,20 +955,51 @@ def on_input_phase(self, input_link):
 
 
 def define_custom_objects_from_file(world: cozmo.world.World, filename: str):
+    # define a unique cube (44mm x 44mm x 44mm) (approximately the same size as a light cube)
+    # with a 30mm x 30mm Diamonds2 image on every face
+    # cube_obj = world.define_custom_cube(CustomObjectTypes.CustomType00,
+    #                                           CustomObjectMarkers.Diamonds2,
+    #                                           44,
+    #                                           30, 30, True)
+
+    # define a unique cube (88mm x 88mm x 88mm) (approximately 2x the size of a light cube)
+    # with a 50mm x 50mm Diamonds3 image on every face
+    # big_cube_obj = world.define_custom_cube(CustomObjectTypes.CustomType01,
+    #                                           CustomObjectMarkers.Diamonds3,
+    #                                           88,
+    #                                           50, 50, True)
+
+
+    # define a unique box (60mm deep x 140mm width x100mm tall)
+    # with a different 30mm x 50mm image on each of the 6 faces
+    # box_obj = world.define_custom_box(cust_type('box', 'box'),
+    #                                         CustomObjectMarkers.Hexagons2,  # front
+    #                                         CustomObjectMarkers.Circles3,   # back
+    #                                         CustomObjectMarkers.Circles4,   # top
+    #                                         CustomObjectMarkers.Circles5,   # bottom
+    #                                         CustomObjectMarkers.Triangles4, # left
+    #                                         CustomObjectMarkers.Triangles3, # right
+    #                                         60, 140, 100,
+    #                                         30, 50, True)
     obj_tree = ET.parse(filename)
     obj_root = obj_tree.getroot()
     custom_objects = []
+    
+    # find attribute.text from node n
+    f = lambda n, attr: n.find(attr).text
```

Comment:
Don't define a function as a lambda, just create a new function. Saving lines is nice, but properly defined functions are easier to read. The instinct to functionalize repeated code is a good one, though!